src/Generator/Generators/CodeGenerator.cs

```diff
@@ -395,6 +395,7 @@ public virtual bool VisitTypeAliasTemplateDecl(TypeAliasTemplate typeAliasTempla
     public static class Helpers
     {
         public static Regex RegexTag = new Regex(@"^(<|</)[a-zA-Z][\w\-]*?>?$");
+        public static Regex RegexCustomTag = new Regex(@"^_\S*");
```

Comment:
Do we really need to include the underscore? What if a custom element is named "thread-safety" i.e. with a dash? I think just \S should do.